Lesson3.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<!--[if lte IE 6]>
+<html class="preIE7 preIE8 preIE9">
+   <![endif]-->
+<!--[if IE 7]>
+   <html class="preIE8 preIE9">
+      <![endif]-->
+<!--[if IE 8]>
+      <html class="preIE9">
+         <![endif]-->
+<!--[if gte IE 9]><!-->
+<html>
+<!--<![endif]-->
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>My ReDi Project</title>
+    <link rel="stylesheet" href="Lesson3.css" type="text/css">
+</head>
+<body>
+    <header>
+        <ul>
+            <li><a href="">HOME</a></li>
+            <li><a href="">ABOUT ME</a></li>
+            <li><a href="index.html" class="current_page">BASE PAGE</a></li>
+        </ul>
+    </header>
+    <div class="main">
+        <h2 class="feel_title">Hi! How are you? </h2>
+        <div class="buttons">
+            <button class="feel_button" id="good_button" width="50px">Good</button>
```

Comment:
camelCase for id's.
another thing - why do you need width here? if you want your buttons to be 50px wide you can put this into your css file. This is important because if later on you would need to change this value you would need to go through your code and change it everywhere. But in css it's just one place.

Reply:
- camelCase for id's - fixed.
- remove width - fixed